crates/bevy_window/src/window.rs

```diff
@@ -199,12 +210,51 @@ impl Window {
         self.physical_height
     }
 
+    /// The window's client position on the x axis in physical pixels.
+    #[inline]
+    pub fn x(&self) -> i32 {
```

Comment:
This seems like as good a time as any to start adopting glam's new "int vec" types. I like the idea of having a single `position() -> IVec2` here (and all other "position" apis added in this pr).

The plan is to slowly start migrating to int vecs where appropriate (ex: individual x/y properties like this and "float vecs" that should be "int vecs")

Reply:
Done.